research-synthesis-agencies.md

```diff
@@ -0,0 +1,68 @@
+
+### What we were researching
+The goal of our research is to inform hypotheses that address the following:
+- How might we create a platform that improves the FOIA requesting experience?
+- How might we make it easier for agencies to respond to requests?
+- How might various FOIA requesters benefit from a national FOIA platform?
+- How might agency processes be improved or enhanced by a national FOIA platform?
+
+### What this document covers
+
+#### Agencies
+- A sample of federal agencies that respond to FOIA requests
+
+### What this document doesn’t cover
+- Requesters
+- Vendors
+
+### What we learned
+
+#### Pain points 
+Government FOIA teams are under-resourced for the level of effort required to respond
+to FOIA requests in a timely manner. Additionally, new or infrequent requesters tend
+to significantly underestimate the level of resources required for fulfilling
+requests, especially broad ones.
+
+#### High
+- A lack of resources makes it difficult to fulfill requests in a timely manner,
+especially for requests that require coordination across bureaucracies or expertise
+for finding records and redacting them.
+- Publishing documents publicly online can require significant additional time and resources, often due to hurdles related to coordinating with separate IT departments and ensuring Section 508 compliance.
+- Current events can cause a spike in requests. These are often extremely broad, resource-intensive requests. For the agencies we spoke with, these requests usually constitute only about 5-10% of all requests, but they take up a majority of time and resources.
+
+#### Medium
+- Some requesters intentionally file extremely broad, vague, or nonspecific FOIA
+requests (e.g., “anything and everything related to…”), but this makes finding
+responsive documents difficult.
+- When faced with broad requests, agencies often attempt to negotiate more finely
+scoped requests that require less resources, which can occasionally result in an
+antagonistic relationship between requesters and FOIA offices. These problematic
+requesters are rare, but often account for a majority of time and resources.
+- Of the agencies we spoke with, most are unhappy with the software tools needed to support FOIA requests.
+- Receiving requests in paper format requires additional manual work.
+
+#### Other learnings
+- Some agencies have frequent requesters who account for a significant proportion of
+their time and resources.
+- Privacy is a concern for moving to a fully electronic submission-based system for
+identification verification (currently done with signature) and document encryption
```

Comment:
I think we should clarify this is done with a "wet signature", because when talking about digital encryption, you might think digital signature.